services/web/server/src/simcore_service_webserver/login/settings.py

```diff
@@ -41,7 +41,7 @@ class LoginSettings(BaseCustomSettings):
     )
 
     LOGIN_2FA_CODE_EXPIRATION_SEC: PositiveInt = Field(
-        default=60, description="Expiration time for code [sec]"
+        default=120, description="Expiration time for code [sec]"
```

Comment:
I suggest unsetting the default entirely and failing if the setting is not given. We should get rid of defaults (in principle, as a code design decision) and always have a comprehensive config handed to the microservices

Reply:
I like your proposition, let us discuss in the Backend if we can do it in more systematic way and remove all defaults :+1: